BaseTools/Source/Python/Workspace/MetaFileParser.py

```diff
@@ -1464,7 +1464,7 @@ def _PostProcess(self):
                                 self._ValueList[0],
                                 self._ValueList[1],
                                 self._ValueList[2],
-                                S1,
+                                self._Scope[0][0],

```

Comment:
Is this change required for this issue?  From line 1405, self._Scope is assigned to [[S1, S2, S3]], so S1 and self._Scope[0][0] should be the same.